EnglishTranslation-master/ETC.tsv

```diff
@@ -4174,17 +4174,17 @@ ETC_20150317_004173	Function is not nagging class:
 ETC_20150317_004174	Your inventory is full. {nl}
 ETC_20150317_004175	Enhancement{/}{/} or
 ETC_20150317_004176	Sell{/}{/}to {nl}key instant shops
-ETC_20150317_004177	A lot of enhancement materials. {nl}
-ETC_20150317_004178	Try item enhancement {/} {/}.
+ETC_20150317_004177	 You have a lot of enhancement materials.{nl}
+ETC_20150317_004178	 Try enhancing an item{/}{/}.
 ETC_20150317_004179	Upgradeable{nl}There are items.{nl}
 ETC_20150317_004180	Use a potion to restore your HP.
 ETC_20150317_004181	{nl} Shortcut: {#0000FF}
 ETC_20150317_004182	Use a potion to recover your SP.
-ETC_20150317_004183	By using a potion, you will be able to recover Stamina
+ETC_20150317_004183	Use a potion to recover your Stamina.
```

Comment:
they're tablets in-game aren't they? does it matter if we still refer to them as potions?